### DISCUSSION TITLE
May I ask how to specify the use of USDT during transactions?

### DISCUSSION THREAD

**Discussion by yunnuowangluo** [WalletConnect/walletconnect-monorepo#3737] [Developer Support] [answered]
May I ask how to specify the use of USDT during transactions?
Thank you

**Comment by glitch-txs** [comment]
could you specify the environment your working on and the libraries you're using? thanks

**Reply by yunnuowangluo** [reply]
I am currently using web3modal, wagmi, walletconnect, and vuejs. I would like to specify using USDT for transactions, but I am not sure how to implement it. Thank you!

**Reply by yunnuowangluo** [reply, marked as answer]
I see. Just use a contract transaction, thank you!
